ci/jenkins/bin/autest.sh

```diff
@@ -16,31 +16,37 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-# Setup autoconf
-cd src
 [ -d tests ] || exit 0
 
-autoreconf -if
-
+# Setup autoconf
 INSTALL="${WORKSPACE}/${BUILD_NUMBER}/install"
+mkdir -p $INSTALL
+cd src
+autoreconf -if
 
 URL="https://ci.trafficserver.apache.org/files/autest"
 AUSB="ausb-${ghprbPullId}.${BUILD_NUMBER}"
 SANDBOX="/var/tmp/${AUSB}"
 
-mkdir -p $INSTALL
-
-./configure --prefix="$INSTALL" \
+./configure --prefix="${INSTALL}" \
             --with-user=jenkins \
             --enable-experimental-plugins \
+            --enable-example-plugins \
             --enable-ccache \
             --enable-debug \
+	    --enable-wccp \
             --enable-werror
 
 # Build and run regressions
-${ATS_MAKE} ${ATS_MAKE_FLAGS} V=1 Q= || exit -1
-${ATS_MAKE} install
-/usr/bin/autest -D ./tests/gold_tests --sandbox "$SANDBOX" --ats-bin "${INSTALL}/bin"
+${ATS_MAKE} -j4 && ${ATS_MAKE} install
+${INSTALL}/bin/traffic_server -K -k -R 1
+[ "0" != "$?" ] && exit -1
+
+# Now run autest
+AUTEST="/usr/bin/autest"
+[ ! -x ${AUTEST} ] && AUTEST="/usr/local/bin/autest"
+
+${AUTEST} -D ./tests/gold_tests --sandbox "$SANDBOX" --ats-bin "${INSTALL}/bin"
```

Comment:
Why don't we use `-j` option to parallelize jobs?

Reply:
You mean -j instead of -j4 ? Because the VMs are not that powerful, and certainly limited in memory, and a -j would make it noticeably slower (maybe even completely fail).

Reply:
I mean `-j` option of AuTest not the option of make. It looks like autest can run in parallel. 
```
$ ./autest.sh -h
...
optional arguments:
  -j JOBS, --jobs JOBS  The number of test to try to run at the same time
...
```

Reply:
Unfortunately, it looks broken :( Sorry for bother you. 
```
Total of 59 test
  Unknown: 59
  Exception: 0
  Failed: 0
  Warning: 0
  Skipped: 0
  Passed: 0
```
When it works, we should turn it on.

/cc @dragon512

Reply:
-J is not working at the moment with AuTest. It is high on my list to finish.